godot-core/src/builtin/projection.rs

```diff
@@ -396,7 +396,13 @@ impl Projection {
     ///
     /// _Godot equivalent: Projection.is_orthogonal()_
     pub fn is_orthogonal(&self) -> bool {
-        self.as_inner().is_orthogonal()
+        self.cols[3].w == 1.0
+
+        // XXX: Test the entire last row?
+        // The argument is that W should not mixed with any other dimensions.
+        // But if the only operation is projection and affine, it suffice
+        // to check if input W is nullified (v33 is zero).
+        // (self.cols[0].w == 0.0) && (self.cols[1].w == 0.0) && (self.cols[2] == 0.0) && (self.cols[3].w == 1.0)
```

Comment:
How does Godot handle this?
In case of doubt, I'd probably adopt its behavior, or what do you think? 🤔

Reply:
It's currently identical to Godot's. I just proposing for maybe a more thorough test (?) But it risk diverging with Godot.

Reply:
I see. Could you maybe change "XXX" to "TODO" and mention what you just replied (that the current impl matches Godot's)?